Models/AnimateAllTheThings/README.md

```diff
@@ -0,0 +1,68 @@
+# Animate All The Things
+
+## Screenshots
+
+## Purpose
+
+This sample model demonstrates the `KHR_animation_pointer` extension which allows animating most properties of a glTF file, in contrast to "regular" animation which can only animate node transforms and morph target weights. These new capabilities bring glTF on par with other powerful animation systems such as USD, Blender, Unity.  
+
+The model serves as a complex integration case to test and ensure which specific extensions and animation capabilities are available in a specific viewer. 
+
+## Support
+
+Known runtimes that support `KHR_animation_pointer` at the time of writing are:  
+
+- [Gestaltor](https://gestaltor.io/)  
+- [Babylon.js](https://sandbox.babylonjs.com/)  
+- [three.js (Needle's fork)](https://three.needle.tools/examples/?q=loader_mu#webgl_loader_multiple)  
+- [Needle Engine](https://needle.tools)  
+
+Known exporters that support `KHR_animation_pointer` at the time of writing are:  
+
+- [UnityGltf (prefrontal cortex' fork)](https://github.com/prefrontalcortex/unitygltf)  
```

Comment:
Blender 4.2.0 and higher can go on this list now.

Reply:
But actually I'm not sure that glTF-Sample-Assets is the correct place to maintain a list of supported importers and exporters, maybe remove this section from the readme.